packages/webapp/src/components/Crop/Management.jsx

```diff
@@ -96,6 +96,11 @@ export default function PureCropManagement({
       {managementPlanCardContents && (
         <CardWithStatusContainer style={{ paddingTop: '16px' }}>
           {filteredManagementPlanCardContents.map((managementPlan, index) => {
+            if (index > 1 && index < 5) {
```

Comment:
Is this a testing artefact? 

Causing some weird behaviours by forcing index:

![Screenshot 2023-07-31 at 10 20 41 AM](https://github.com/LiteFarmOrg/LiteFarm/assets/30075973/44a8612a-0ba5-45e6-b3b6-35fb8feed550)

Reply:
Nice catch. Removed.